scripts.py

```diff
@@ -1011,18 +1010,10 @@ def do_run(self, ignore_search=False):
                 # Elasticsearch isn't configured, so do nothing.
                 pass
 
-        # Set a timestamp that represents the new database's version.
```

Comment:
For an existing database, that has the `libraries` table, but doesn't have an alembic stamp, won't we miss setting the version here?

Reply:
Yes that's right, from the original intent of the script, this is meant only to "initialise a brand new database".
The total life cycle depends on the `02_manage_simplified_database.sh` script running in 'auto' mode.
First the `initialize_database` (this script) will run, then the `core/bin/migrate_database` will run.
`migrate_database` does an `upgrade head` which is idempotent and will create an `alembic_version` table if it is missing.

Reply:
ONLY if we are an empty, brand new DB should we `stamp head` since SQLAlchemy will autocreate all tables and constraints.
If we are a non-empty DB without an alembic_version, then `upgrade head` will create the table and run all migrations in the repository. This should only happen during the FIRST run after deploying the alembic changes, thereafter the DB with a head stamp should exist.
Is there a case where we have a non-empty database and we DO NOT want to run the alembic revisions?

Reply:
No, this makes sense to me. Thank you for the clarification.